request-header-parser.js

```diff
@@ -0,0 +1,15 @@
+const Http = require("http");
```

Comment:
you have created this program but this will only run locally. You need to create a npm module out of it.